config/jenkins_release.ci.jenkins.io.yaml

```diff
@@ -430,6 +430,9 @@ controller:
           disabledAdministrativeMonitors:
             - "jenkins.security.QueueItemAuthenticatorMonitor"
         security:
+          contentSecurityPolicyConfiguration:
+            reportOnly: true
+            rule: "default-src 'self'; style-src 'self' 'unsafe-inline'; img-src 'self' data: avatars.githubusercontent.com; media-src 'self' data: ; script-src 'self' 'report-sample' usage.jenkins.io;"
```

Comment:
I don't believe this config is right, see https://github.com/jenkins-infra/jenkins-infra/pull/4376/files#r2315523711

Reply:
O thanks @timja , will wait for @daniel-beck reply then